examples/Visualization with plato.ipynb

```diff
@@ -14,53 +14,70 @@
    "metadata": {},
```

Comment:
<p>and here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>